src/rpc/blockchain.cpp

```diff
@@ -1605,33 +1605,49 @@ UniValue getblockchaininfo(const JSONRPCRequest &request) {
+                                                 {RPCResult::Type::NUM, "round_size",
+                                                  "number of block per round of voting"},
+                                                 {RPCResult::Type::NUM, "voting_period",
+                                                  "Number of rounds required for voting threshold check"},
+                                                 {RPCResult::Type::OBJ, "miners",
+                                                  "miner numeric statistics about RIP1 signalling for a softfork",
                                                   {
-                                                          {RPCResult::Type::NUM, "period",
-                                                           "the length in blocks of the BIP9 signalling period"},
+                                                          {RPCResult::Type::NUM, "mean_percentage",
+                                                           "the mean approved percentage for this current voting period"},
+                                                          {RPCResult::Type::NUM, "weighted_yes",
+                                                           "the number approved votes for this voting period"},
+                                                          {RPCResult::Type::NUM, "weight",
+                                                           "the number votes for this voting period"},
+                                                          {RPCResult::Type::NUM, "samples",
+                                                           "the number round has been passed"},
                                                           {RPCResult::Type::NUM, "threshold",
-                                                           "the number of blocks with the version bit set required to activate the feature"},
-                                                          {RPCResult::Type::NUM, "elapsed",
-                                                           "the number of blocks elapsed since the beginning of the current period"},
-                                                          {RPCResult::Type::NUM, "count",
-                                                           "the number of blocks with the version bit set in the current period"},
-                                                          {RPCResult::Type::BOOL, "possible",
-                                                           "returns false if there are not enough blocks left in this period to pass activation threshold"},
+                                                           "current pass activation threshold"},
+                                                          {RPCResult::Type::BOOL, "approved",
+                                                           "returns false if majority of miners are not yet approved"},
```

Comment:
majority means > 50%, so we can't use that.
```suggestion
                                                           "returns false if miners have not yet approved"},
```